wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearningSetup/ActiveLearningInitialLabels.vue

```diff
@@ -498,6 +500,11 @@ export default Vue.extend({
               {{ imageNamesById[imageId] }}
             </option>
           </select>
+          <annotation-opacity-control
+            :active-learning-setup="true"
+            :categories="allNewCategories"
+            :update="synchronizeCategories"
+          />
```

Comment:
I think we should also pass the opacity of the current image's label annotation. Currently it looks like if I modify the opacity with the slider then refresh, my modified opacity is saved, but the slider resets itself at the max position.

Reply:
This is fixed with the use of the config file

Reply:
This is fixed with the use of the config file